e2e-tests/non-voting/conf/nonvoting-rs0.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: nonvoting
+spec:
+  #platform: openshift
+  image:
+  mongod:
+    security:
+      enableEncryption: true
+  replsets:
+  - name: rs0
+    affinity:
+      antiAffinityTopologyKey: none
+    nonvoting:
+      enabled: true
+      size: 3
+      affinity:
+        antiAffinityTopologyKey: "kubernetes.io/hostname"
+        advanced:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchLabels:
+                    app.kubernetes.io/component: nonVoting
+                    app.kubernetes.io/instance: nonvoting
+                    app.kubernetes.io/managed-by: percona-server-mongodb-operator
+                    app.kubernetes.io/name: percona-server-mongodb
+                    app.kubernetes.io/part-of: percona-server-mongodb
+                    app.kubernetes.io/replset: rs0
+                topologyKey: kubernetes.io/hostname
+      resources:
+        limits:
+          cpu: "300m"
+          memory: "0.5G"
+        requests:
+          cpu: "200m"
+          memory: "0.3G"
+      volumeSpec:
+        persistentVolumeClaim:
+          resources:
+            requests:
+              storage: 2Gi
+      containerSecurityContext:
+        privileged: false
+      podSecurityContext:
+        fsGroup: 1001
+        supplementalGroups: [1001, 1002, 1003]
```

Comment:
as I can see podSecurityContext is not passed, also I am worried that Configuration, ReadinessProbe, LivenessProbe, ContainerSecurityContext can be not passed